templates/visuals/.api/v1.5.0/PowerBI-visuals.d.ts

```diff
@@ -956,73 +956,73 @@ declare module powerbi {
     /** Describes a data value type in the client type system. Can be used to get a concrete ValueType instance. */
     export interface ValueTypeDescriptor {
         // Simplified primitive types
-        readonly text?: boolean;
```

Comment:
It can't be compiled by TypeScript [1.8.10](https://github.com/AviSander/PowerBI-visuals-tools/blob/master/package.json#L39). We should use 2.x.x if we want to use the readonly property.